src/hardware/sblaster.cpp

```diff
@@ -738,15 +738,25 @@ static void PlayDMATransfer(uint32_t bytes_requested)
 			bytes_read = ReadDMA8(bytes_to_read, sb.dma.remain_size);
 			samples = bytes_read + sb.dma.remain_size;
 			frames = check_cast<uint16_t>(samples / channels);
-			if (sb.dma.sign) {
-				sb.chan->AddSamples_s8s(frames,
-				         maybe_silence(samples,
-				                       reinterpret_cast<int8_t *>(sb.dma.buf.b8)));
-			} else {
-				sb.chan->AddSamples_s8(frames,
-				         maybe_silence(samples, sb.dma.buf.b8));
+
+			// Only add whole frames when in a stereo DMA mode. The
+			// number of frames comes from the DMA request and
+			// threfore is data from user-space.
+			LOG_MSG("DMA8 stereo: %d frames", frames);
```

Comment:
Should this be debug level?

Reply:
Accidentally left that in; knocking out. Thanks!